src/api/app/policies/token_policy.rb

```diff
@@ -45,4 +45,8 @@ def webui_trigger?
   def show?
     webui_trigger?
   end
+
+  def workflow_runs_index?
```

Comment:
In [Webui::Users::TokensController](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/users/tokens_controller.rb), the tokens are already scoped to the logged-in user. I don't see a need to redo a policy check for this. The link can be displayed if the token is a `Token::Workflow` and if somehow a user tries to access the workflow runs page for a non-workflow token, `WorkflowRunPolicy` will prevent that.

Reply:
You are right, no need to double check, I removed the policy and only check for the token type in order to show the link only for the workflow tokens